content/docs/notebooks/troubleshoot.md

```diff
@@ -0,0 +1,74 @@
++++
+title = "Troubleshooting Guide"
+description = "Fixing common problems in Kubeflow notebooks"
+weight = 50
++++
+
+## Persistent Volumes and Persistent Volumes Claims
+
+First, make sure that PVCs are bounded when using jupter notebooks. This should
+not be a problem when using managed Kuberenets. But if you are using Kubernetes
+on-prem, checkout [this](https://www.kubeflow.org/docs/use-cases/kubeflow-on-multinode-cluster/) guide.
```

Comment:
It's tech writing best practice to give people a heads up where a link will take them, rather than using just the word "this".

So, change this:
```
[this](https://www.kubeflow.org/docs/use-cases/kubeflow-on-multinode-cluster/) guide.
```

to this:

```
the guide to [Kubeflow on-prem in a multi-node Kubernetes cluster](/docs/use-cases/kubeflow-on-multinode-cluster/).
```